packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -66,6 +70,18 @@ const getAppEntrypoint = () => {
     return p.join(process.cwd(), 'app', 'ssr.js')
 }
 
+const validateConfiguration = () => {
```

Comment:
Much better.. a few nits.

1. lets get a block comment here, particularly calling out that we are going to be doing app validation here both config (future) and extension config
2. I think `validateAppConfiguration` might be a little more intentional in it's name, especially with the current content that is only doing extensibility work.
3. I'm not a fan how we are using 2 utilities from the extensibility lib and using one to prepare args for another. Could we simply make the `validateExtensionDependencies` accept the raw form of the extensions configuration value as well as the expanded entries?